### PR TITLE
[Accessibility] Implement Xcode 16.2 beta 1-3 changes.

### PR DESCRIPTION
**Pull request by rolfbjarne**

### CODE REVIEW

File: src/accessibility.cs [hunk 1]
```diff
@@ -406,4 +406,229 @@ enum AXTechnology {
...
+	}
+
+	[TV (18, 2), Mac (15, 2), iOS (18, 2), MacCatalyst (18, 2)]
+	[BaseType (typeof (NSObject))]
+	interface AXMathExpression {
+	}
+
+	[TV (18, 2), Mac (15, 2), iOS (18, 2), MacCatalyst (18, 2)]
+	[BaseType (typeof (AXMathExpression))]
+	interface AXMathExpressionNumber {
```
Comment (dalexsoto) [inline src/accessibility.cs:459]: This has a ctor and properties are readonly

```suggestion
	[BaseType (typeof (AXMathExpression))]
	[DisableDefaultCtor]
	interface AXMathExpressionNumber {
```

File: src/accessibility.cs [hunk 2]
```diff
@@ -406,4 +406,229 @@ enum AXTechnology {
...
+		[Export ("initWithContent:")]
+		NativeHandle Constructor (string content);
+
+		[Export ("content")]
+		string Content { get; }
+	}
+
+	[TV (18, 2), Mac (15, 2), iOS (18, 2), MacCatalyst (18, 2)]
+	[BaseType (typeof (AXMathExpression))]
+	interface AXMathExpressionIdentifier {
```
Comment (dalexsoto) [inline src/accessibility.cs:469]: Same, this has a ctor and properties are readonly

```suggestion
	[BaseType (typeof (AXMathExpression))]
	[DisableDefaultCtor]
	interface AXMathExpressionIdentifier {
```

File: src/accessibility.cs [hunk 3]
```diff
@@ -406,4 +406,229 @@ enum AXTechnology {
...
+		[Export ("initWithContent:")]
+		NativeHandle Constructor (string content);
+
+		[Export ("content")]
+		string Content { get; }
+	}
+
+	[TV (18, 2), Mac (15, 2), iOS (18, 2), MacCatalyst (18, 2)]
+	[BaseType (typeof (AXMathExpression))]
+	interface AXMathExpressionOperator {
```
Comment (dalexsoto) [inline src/accessibility.cs:479]: Same

```suggestion
	[BaseType (typeof (AXMathExpression))]
	[DisableDefaultCtor]
	interface AXMathExpressionOperator {
```

File: src/accessibility.cs [hunk 4]
```diff
@@ -406,4 +406,229 @@ enum AXTechnology {
...
+		[Export ("initWithContent:")]
+		NativeHandle Constructor (string content);
+
+		[Export ("content")]
+		string Content { get; }
+	}
+
+	[TV (18, 2), Mac (15, 2), iOS (18, 2), MacCatalyst (18, 2)]
+	[BaseType (typeof (AXMathExpression))]
+	interface AXMathExpressionText {
```
Comment (dalexsoto) [inline src/accessibility.cs:489]: Same
```suggestion
	[BaseType (typeof (AXMathExpression))]
	[DisableDefaultCtor]
	interface AXMathExpressionText {
```

File: src/accessibility.cs [hunk 5]
```diff
@@ -406,4 +406,229 @@ enum AXTechnology {
...
+		[Export ("initWithContent:")]
+		NativeHandle Constructor (string content);
+
+		[Export ("content")]
+		string Content { get; }
+	}
+
+	[TV (18, 2), Mac (15, 2), iOS (18, 2), MacCatalyst (18, 2)]
+	[BaseType (typeof (AXMathExpression))]
+	interface AXMathExpressionFenced {
```
Comment (dalexsoto) [inline src/accessibility.cs:499]: Same
```suggestion
	[BaseType (typeof (AXMathExpression))]
	[DisableDefaultCtor]
	interface AXMathExpressionFenced {
```

File: src/accessibility.cs [hunk 6]
```diff
@@ -406,4 +406,229 @@ enum AXTechnology {
...
+		[Export ("openString")]
+		string OpenString { get; }
+
+		[Export ("closeString")]
+		string CloseString { get; }
+	}
+
+	[TV (18, 2), Mac (15, 2), iOS (18, 2), MacCatalyst (18, 2)]
+	[BaseType (typeof (AXMathExpression))]
+	interface AXMathExpressionRow {
```
Comment (dalexsoto) [inline src/accessibility.cs:515]: Same

```suggestion
	[BaseType (typeof (AXMathExpression))]
	[DisableDefaultCtor]
	interface AXMathExpressionRow {
```

File: src/accessibility.cs [hunk 7]
```diff
@@ -406,4 +406,229 @@ enum AXTechnology {
...
+		[Export ("initWithExpressions:")]
+		NativeHandle Constructor (AXMathExpression [] expressions);
+
+		[Export ("expressions")]
+		AXMathExpression [] Expressions { get; }
+	}
+
+	[TV (18, 2), Mac (15, 2), iOS (18, 2), MacCatalyst (18, 2)]
+	[BaseType (typeof (AXMathExpression))]
+	interface AXMathExpressionTable {
```
Comment (dalexsoto) [inline src/accessibility.cs:525]: Same

```suggestion
	[BaseType (typeof (AXMathExpression))]
	[DisableDefaultCtor]
	interface AXMathExpressionTable {
```

File: src/accessibility.cs [hunk 8]
```diff
@@ -406,4 +406,229 @@ enum AXTechnology {
...
+		[Export ("initWithExpressions:")]
+		NativeHandle Constructor (AXMathExpression [] expressions);
+
+		[Export ("expressions")]
+		AXMathExpression [] Expressions { get; }
+	}
+
+	[TV (18, 2), Mac (15, 2), iOS (18, 2), MacCatalyst (18, 2)]
+	[BaseType (typeof (AXMathExpression))]
+	interface AXMathExpressionTableRow {
```
Comment (dalexsoto) [inline src/accessibility.cs:535]: Same

```suggestion
	[BaseType (typeof (AXMathExpression))]
	[DisableDefaultCtor]
	interface AXMathExpressionTableRow {
```

File: src/accessibility.cs [hunk 9]
```diff
@@ -406,4 +406,229 @@ enum AXTechnology {
...
+		[Export ("initWithExpressions:")]
+		NativeHandle Constructor (AXMathExpression [] expressions);
+
+		[Export ("expressions")]
+		AXMathExpression [] Expressions { get; }
+	}
+
+	[TV (18, 2), Mac (15, 2), iOS (18, 2), MacCatalyst (18, 2)]
+	[BaseType (typeof (AXMathExpression))]
+	interface AXMathExpressionTableCell {
```
Comment (dalexsoto) [inline src/accessibility.cs:545]:
```suggestion
	[BaseType (typeof (AXMathExpression))]
	[DisableDefaultCtor]
	interface AXMathExpressionTableCell {
```

File: src/accessibility.cs [hunk 10]
```diff
@@ -406,4 +406,229 @@ enum AXTechnology {
...
+		[Export ("initWithExpressions:")]
+		NativeHandle Constructor (AXMathExpression [] expressions);
+
+		[Export ("expressions")]
+		AXMathExpression [] Expressions { get; }
+	}
+
+	[TV (18, 2), Mac (15, 2), iOS (18, 2), MacCatalyst (18, 2)]
+	[BaseType (typeof (AXMathExpression))]
+	interface AXMathExpressionUnderOver {
```
Comment (dalexsoto) [inline src/accessibility.cs:555]: Same

```suggestion
	[BaseType (typeof (AXMathExpression))]
	[DisableDefaultCtor]
	interface AXMathExpressionUnderOver {
```

File: src/accessibility.cs [hunk 11]
```diff
@@ -406,4 +406,229 @@ enum AXTechnology {
...
+		[Export ("underExpression")]
+		AXMathExpression UnderExpression { get; }
+
+		[Export ("overExpression")]
+		AXMathExpression OverExpression { get; }
+	}
+
+	[TV (18, 2), Mac (15, 2), iOS (18, 2), MacCatalyst (18, 2)]
+	[BaseType (typeof (AXMathExpression))]
+	interface AXMathExpressionSubSuperscript {
```
Comment (dalexsoto) [inline src/accessibility.cs:571]:
```suggestion
	[BaseType (typeof (AXMathExpression))]
	[DisableDefaultCtor]
	interface AXMathExpressionSubSuperscript {
```

File: src/accessibility.cs [hunk 12]
```diff
@@ -406,4 +406,229 @@ enum AXTechnology {
...
+		[Export ("subscriptExpressions")]
+		AXMathExpression [] SubscriptExpressions { get; }
+
+		[Export ("superscriptExpressions")]
+		AXMathExpression [] SuperscriptExpressions { get; }
+	}
+
+	[TV (18, 2), Mac (15, 2), iOS (18, 2), MacCatalyst (18, 2)]
+	[BaseType (typeof (AXMathExpression))]
+	interface AXMathExpressionFraction {
```
Comment (dalexsoto) [inline src/accessibility.cs:587]: Same

```suggestion
	[BaseType (typeof (AXMathExpression))]
	[DisableDefaultCtor]
	interface AXMathExpressionFraction {
```

File: src/accessibility.cs [hunk 13]
```diff
@@ -406,4 +406,229 @@ enum AXTechnology {
...
+		[Export ("numeratorExpression")]
+		AXMathExpression NumeratorExpression { get; }
+
+		[Export ("denimonatorExpression")]
+		AXMathExpression DenimonatorExpression { get; }
+	}
+
+	[TV (18, 2), Mac (15, 2), iOS (18, 2), MacCatalyst (18, 2)]
+	[BaseType (typeof (AXMathExpression))]
+	interface AXMathExpressionMultiscript {
```
Comment (dalexsoto) [inline src/accessibility.cs:600]: Same

```suggestion
	[BaseType (typeof (AXMathExpression))]
	[DisableDefaultCtor]
	interface AXMathExpressionMultiscript {
```

File: src/accessibility.cs [hunk 14]
```diff
@@ -406,4 +406,229 @@ enum AXTechnology {
...
+		[Export ("prescriptExpressions")]
+		AXMathExpressionSubSuperscript [] PrescriptExpressions { get; }
+
+		[Export ("postscriptExpressions")]
+		AXMathExpressionSubSuperscript [] PostscriptExpressions { get; }
+	}
+
+	[TV (18, 2), Mac (15, 2), iOS (18, 2), MacCatalyst (18, 2)]
+	[BaseType (typeof (AXMathExpression))]
+	interface AXMathExpressionRoot {
```
Comment (dalexsoto) [inline src/accessibility.cs:616]: Same
```suggestion
	[BaseType (typeof (AXMathExpression))]
	[DisableDefaultCtor]
	interface AXMathExpressionRoot {
```

**Review by dalexsoto — CHANGES_REQUESTED** [review]

**Review by haritha-mohan — APPROVED** [review]
after Alex's comments 👍

File: src/accessibility.cs [hunk 15]
```diff
@@ -406,4 +406,229 @@ enum AXTechnology {
...
+		[return: NullAllowed]
+		AXFeatureOverrideSession BeginOverrideSession (AXFeatureOverrideSessionOptions enableOptions, AXFeatureOverrideSessionOptions disableOptions, [NullAllowed] out NSError error);
+
+		[Export ("endOverrideSession:error:")]
+		bool EndOverrideSession (AXFeatureOverrideSession session, [NullAllowed] out NSError error);
+	}
+
+	[TV (18, 2), Mac (15, 2), iOS (18, 2), MacCatalyst (18, 2)]
+	[BaseType (typeof (NSObject))]
+	interface AXMathExpression {
```
Comment (rolfbjarne) [inline src/accessibility.cs:454]:
```suggestion
	[BaseType (typeof (NSObject))]
	[DisableDefaultCtor]
	interface AXMathExpression {
```

**Review by dalexsoto — APPROVED** [review]

**Pull request merged**